lib/neat/version.rb

```diff
@@ -1,3 +1,3 @@
 module Neat
-  VERSION = '1.6.0'
+  VERSION = '1.7.1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.